requirements.txt

```diff
@@ -9,39 +9,39 @@
 aniso8601==6.0.0          # via graphene
 certifi==2019.6.16        # via requests, sentry-sdk
 chardet==3.0.4            # via requests
-coreapi==2.3.3
+coreapi==2.3.3            # via -r requirements.in
 coreschema==0.0.4         # via coreapi
 defusedxml==0.6.0         # via odfpy, python3-openid
 diff-match-patch==20181111  # via django-import-export
-django-admin-sortable==2.2.3
-django-allauth==0.39.1
+django-admin-sortable==2.2.3  # via -r requirements.in
+django-allauth==0.39.1    # via -r requirements.in
 django-anymail==6.1.0     # via django-ilmoitin
-django-cors-headers==3.0.2
-django-enumfields==1.0.0
-django-environ==0.4.5
-django-filter==2.2.0
-django-graphql-jwt==0.2.2
-django-guardian==2.1.0
-django-helusers==0.4.5
-django-ilmoitin==0.2.0
-django-import-export==2.0.2
+django-cors-headers==3.0.2  # via -r requirements.in
+django-enumfields==1.0.0  # via -r requirements.in
+django-environ==0.4.5     # via -r requirements.in
+django-filter==2.2.0      # via -r requirements.in
+django-graphql-jwt==0.2.2  # via -r requirements.in
+django-guardian==2.1.0    # via -r requirements.in
+django-helusers==0.4.5    # via -r requirements.in
+django-ilmoitin==0.2.0    # via -r requirements.in
+django-import-export==2.0.2  # via -r requirements.in
 django-js-asset==1.2.2    # via django-mptt
 django-mailer==1.2.6      # via django-ilmoitin
 django-mptt==0.10.0       # via django-munigeo
-django-munigeo==0.3.3
+django-munigeo==0.3.5     # via -r requirements.in
```

Comment:
This was the only real change.  :confused:

Reply:
Good catch 👍